src/cache-updater/Dockerfile.cacheUpdater

```diff
@@ -0,0 +1,49 @@
+FROM python:3.11-bullseye as python-builder
+# We intentionally do not set a default value 
+# because we want to provide a specific snooty parser
+# version every time.
+ARG SNOOTY_PARSER_VERSION
+WORKDIR /usr/app
+
+RUN apt-get -o Acquire::Check-Valid-Until=false update
+RUN apt-get -y install git zip unzip tar make curl
+RUN python3 -m pip install poetry
+
+# install snooty parser
+RUN git clone -b v${SNOOTY_PARSER_VERSION} --depth 1 https://github.com/mongodb/snooty-parser.git \
+    && cd snooty-parser \
+    && python3 -m poetry install \
+    && make package \
+    && mv dist/snooty /opt/
+
+FROM node:18-bullseye-slim as builder
+WORKDIR /usr/app
+
+COPY package*.json ./
+RUN npm ci
+
+COPY tsconfig*.json ./
+COPY api api/
+COPY src src/
+
+RUN npm run build:esbuild:cacheUpdater
+
+FROM node:18-bullseye-slim
+WORKDIR /usr/app
+
+
+ENV PATH="${PATH}:/opt/snooty"
+RUN apt-get -o Acquire::Check-Valid-Until=false update && apt-get -y install git
+
+
+
+COPY --from=python-builder /opt/ /opt/
+COPY --from=builder /usr/app/dist/ .
+
+RUN useradd -ms /bin/bash docsworker && \
+    chmod 755 -R /usr/app  && \
+    chown -Rv docsworker /usr/app
+
+USER docsworker
```

Comment:
Just wondering why this is `docsworker` instead of `docsworker-xlarge`. I remember us having to change this for the local autobuilder to `docsworker-large` to keep consistent.

Reply:
I think I must've copied this from a previous version of the `Dockerfile.local`. It should be fine in this cause because it needs to be consistent within the Autobuilder because we use the username within the Autobuilder code. The code in this Dockerfile doesn't depend on the username at all, we just need a random user to place the AWS credentials.